packages/components/src/toolbar-button/style.scss

```diff
@@ -1,3 +1,16 @@
+.components-toolbar-button {
+	padding: 0 9px;
+	border-radius: $radius-round-rectangle;
+
+	&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover {
+		@include button-style__hover;
+	}
+
+	&:not(:disabled):not([aria-disabled="true"]):not(.is-default):active {
+		@include button-style__active;
+	}
+}
+
```

Comment:
I've added some `IconButton` styles here, since `components-toolbar-button` is just `Button`, which doesn't have the border styles on hover and active. But maybe that should be a `Button` state that `IconButton` (and therefore any toolbar button) inherits?

Reply:
This is how the toolbar button with text only looks like at the moment:
![Screen Shot 2019-12-06 at 07 45 37](https://user-images.githubusercontent.com/699132/70302087-7330d900-17fc-11ea-9e4c-b02dae93229b.png)

I think it will change in the future given that there is this ongoing exploration of how to redefine the look and behavior of the block toolbar. That's why I would say, as long as it looks the same, it doesn't matter how we approach it :)

Reply:
Well, I nearly removed styles for the toolbar button in #17748.

I brought them back in https://github.com/WordPress/gutenberg/pull/17748/commits/a282f319b05ebf5af597cb839c0e7455c711ddec.

We definitely should find a better way to handle them. I guess your proposal is closer to what we need as it allows to easily style the button without an icon.

Reply:
I landed #17748 so you might want to update PR with whatever I changed there :)

Reply:
In addition, @youknowriad opened PR which brings more consistency to all types of buttons in https://github.com/WordPress/gutenberg/pull/19058.

Reply:
I removed these styles since they've been already added to `Button` on #19058

This is a generic toolbar button. If we abstract the tab button into a new component (`Tab`, `TabButton`?), we may have a `ToolbarTab`, `ToolbarTabButton`.